lambdas/service/app.py

```diff
@@ -1348,8 +1348,7 @@ def _dss_files(uuid, fetch=True):
 
 def file_url(uuid: str, fetch: bool = True, **params: str):
     uuid = urllib.parse.quote(uuid, safe="")
-    view_function = fetch_dss_files if fetch else dss_files
-    url = self_url(endpoint_path=view_function.path.format(uuid=uuid))
+    url = self_url(endpoint_path=f'{"/fetch" if fetch else ""}/dss/files/{uuid}')
```

Comment:
I remember now that I added `.path` so we don't have to duplicate the URL here. We need revert this change.